Add aws release 16.2.0 (#876)
giantswarm/releases: aws/v16.2.0/README.md

# :zap: Giant Swarm Release v16.2.0 for AWS :zap:

> This is a security release featuring the latest Flatcar Linux version as well as an updated version of the Giant Swarm applications.

Reviewer: ```suggestion
This is a security release featuring the latest Flatcar Linux version as well as various updates to Giant Swarm applications.
```
